.chloggen/rm-attributes-gate.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: service
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove the `telemetry.newPipelineTelemetry` feature gate.
+
+# One or more tracking issues or pull requests related to the change
+issues: [12856]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

I think it would be good to add a bit more context since it's a breaking change:
```suggestion
subtext: |
  The feature is now enabled by default and can no longer be disabled. Please refer to the v0.123.0 changelog for a description of the gate.
  
  This incidentally fixes the issue introduced in that version where logs on standard output no longer contained any component identifiers when the gate was left disabled.
```